docs/GettingStarted.md

```diff
@@ -64,7 +64,8 @@ of the Datadog tracer, you can override the following defaults:
       tracer: Datadog.tracer,
       debug: false,
       trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126
+      trace_agent_port: 8126,
+      env: Rails.env
```

Comment:
Yes, I think. (minus the little nitpicks I left, concerning what is more importante, `:env` or `:tags['env']`?)